addons/event/report/event_event_templates.xml

```diff
@@ -1,108 +1,135 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<!-- EVENT FOLDABLE BADGE -->
+<!-- EVENT BADGE CARD -->
+
+<template id="event_report_template_badge_card">
```

Comment:
As it is a tool template, would move at bottom of the file maybe, to keep the main ones on top ?